semgrep/semgrep/evaluation.py

```diff
@@ -11,13 +11,12 @@
 from typing import Tuple
 
 logger = logging.getLogger(__name__)
-# disable logging from this module most of the time
```

Comment:
I think this may have been unintentionally left in. It makes it difficult to debug this module because it's impossible to set the logger level to `DEBUG` with this here. Now that we have the `--debug` flag I think this is no longer necessary, or unintentional.

Reply:
I did it intentionally -- it was really difficult to debug other parts of the program with the large amount of output this module spews. But very open to other approaches

Reply:
Hmm, I usually try to err on the side of over-outputting with debug output vs. under-outputting. It's much easier to filter existing output than include additional output information in an already released version. If you're looking for a specific message in the debug output you can always:

```
$ semgrep --debug ... 2>&1 | grep 'compiled result'
```

Or, if you want to squelch a particularly noisy log line you can do something like:

```
$ semgrep --debug  ... 2>&1 | grep -v 'after filter'
```

Another good option here is including `%(name)s` in our [logger format](https://docs.python.org/3/library/logging.html#logrecord-attributes) in `semgrep/semgrep/util.py`. Then we could do something like the following to squelch a single module's logging:

```
$ semgrep --debug  ... 2>&1 | grep -v 'semgrep.evaluation'
```

Reply:
I now use in semgrep-core a logging library inspired by the python one, and you can use a config file to specify the different levels for the different modules outside the code. See https://github.com/returntocorp/semgrep/blob/develop/semgrep-core/log_config.json.ex1